### PR TITLE
Spotify auth

### PR DESCRIPTION
**Pull request by Sparshith**

### CODE REVIEW

File: cmd/songskiweb/main.go [hunk 1]
```diff
@@ -1,43 +1,39 @@
...
 
 	appConfig := config.Initialize()
 	log.Info("App config loaded successfully..")
 
 	mysqlDB, err := mysql.GetDB(appConfig.MySQL)
 	if err != nil {
-		log.Error("Mysql failed to start")
+		log.Error(err)
+	}
+	if err == nil {
```
Comment (Sparshith) [inline cmd/songskiweb/main.go]: This check is not required, we are already checking for err != nil above

File: cmd/songskiweb/main.go [hunk 2]
```diff
@@ -1,43 +1,39 @@
...
 	http.HandleFunc("/auth/spotify/callback", auth.SpotifyHandleCallback)
 	http.HandleFunc("/spotify/logout", auth.SpotifyLogout)
+	http.ListenAndServe(":6911", nil)
 
-	http.HandleFunc("/soundcloud/login", songskiRouter.SoundCloudLogin)
-	http.HandleFunc("/auth/soundcloud", auth.SoundCloudAuth)
-	http.HandleFunc("/auth/soundcloud/callback", auth.SoundCloudHandleCallback)
-	http.HandleFunc("/soundcloud/logout",auth.SoundCloudLogout)
-
-	log.Info("Starting HTTP Server on port 3000..")
```
Comment (Sparshith) [inline cmd/songskiweb/main.go]: Add this log back

File: pkg/models/userToken.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+)
+
+type UserAuth struct {
+	ID           string
+	AccessToken  string
+	ExpiresAt    time.Time
+	RefreshToken string
+}
+
+func InsertUserCredentials(ut *UserAuth, MysqlDB *gorm.DB) {
```
[the remark this answers is not in the document]
Comment (nikhiladharmaji) [reply pkg/models/userToken.go]: yeah that makes so much more sense. will do

File: pkg/models/userToken.go [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+	MysqlDB.AutoMigrate(UserAuth{})
+	MysqlDB.Debug().Where(UserAuth{ID: ut.ID}).Assign(UserAuth{AccessToken: ut.AccessToken, RefreshToken: ut.RefreshToken, ExpiresAt: ut.ExpiresAt}).FirstOrCreate(&ut)
```
[the remark this answers is not in the document]
Comment (nikhiladharmaji) [reply pkg/models/userToken.go]: Prints the MYSQL output to console eg: "Query successful, 1 row affected"

File: pkg/models/userToken.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (nikhiladharmaji) [reply pkg/models/userToken.go]: Doneski

**Pull request merged**